src/app/find/questionFlow/QuestionFlow.jsx

```diff
@@ -123,7 +124,7 @@ class QuestionFlow extends Component {
             <div className="px-2 mb-5">
               {this.renderQuestion()}
               <div className="p-3 mx-2 fixed-bottom">
-                <Button onClick={skipQuestion} secondary fluid className="position-relative">
+                <Button onClick={skipQuestion} fluid className="position-relative" className="Button-terciary">
```

Comment:
Oh, and also you have className twice here. Not sure what happens - maybe the 2nd overrides the first?
Anyway, this would need to be consolidated into a single className string with a space between the classes.

(Though this might be a moot point considering the previous comment.)

Reply:
I'm lost here. Can we go over this together?

Reply:
Yup, I think my first comment would be a good subject to talk about in real time.
As for the second, it's just that I don't think having 2 "className" attributes works properly, so it should be merged (e.g. `<Button onClick={skipQuestion} fluid className="position-relative Button-terciary" >`).